### DISCUSSION TITLE
Show errors on the map

### DISCUSSION THREAD

**Discussion by stejo388** [Thomas55555/husqvarna_automower#477] [Ideas]
It would be great to show when and where an error occurs on the map. After a while you could then see patters where you need to fix something in the garden

**Comment by Din-BH** [comment]
You can make that happen as is. Use the camera as image in a picture-elements card and set a conditional state-label showing the problem in case of an error.

**Reply by stejo388** [reply]
Hi,

I´m not sure how to get that to work. What I would like to have is a map where you basically could filter different error and time period to show them on the map. Not just when a error occurs.

**Comment by Din-BH** [comment]
Hmm. While I am sure it would be super useful to iron out issues with your lawn cabling etc, it sounds to me like quite a complex use case for a HA integration. Maybe more fitted for the Husqvarna app. Maybe you can contact them with the suggestion?
